docs/api-references/counts.md

```diff
@@ -0,0 +1,29 @@
+## Counts - API
+
+### `GET` List Count Value
+```
+/api/counts
+```
+Get all object and its count value
```

Comment:
It not `Get all object`. Just count for all resource. Or you can make more clear like this: **Count data for Category, DailyMenu, Food, Material, Order, OrderItem, Supplier, User**